modules/core/src/main/scala/higherkindness/droste/basis.scala

```diff
@@ -42,6 +42,9 @@ trait Project[F[_], R] { self =>
       F: Foldable[F]): U =
     Project.collect[F, R, U, B](r)(pf)
 
+  def find[B](r: R)(pf: PartialFunction[R, B])(implicit F: Foldable[F]): Option[B] =
```

Comment:
I'd prefer the Boolean variation, as it's more consistent with find methods in the Scala stdlib.